src/Analysis/GWASV2/Shared/AttritionTableWrapper/AttritionTable/AttritionTableRow.jsx

```diff
@@ -0,0 +1,158 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useQuery } from 'react-query';
+import { Spin } from 'antd';
+import {
+  fetchConceptStatsByHareSubset,
+  // fetchConceptStatsByHareSubsetCC,
+  queryConfig,
+} from '../../wizardEndpoints/cohortMiddlewareApi';
+import BarChart from '../ChartIcons/BarChart';
+import EulerDiagram from '../ChartIcons/EulerDiagram';
+
+const AttritionTableRow = ({
+  selectedCohort,
+  rowType,
+  rowObject,
+  outcome,
+  currentCovariateAndCovariatesFromPrecedingRows,
+  sourceId,
+}) => {
+  const [breakdownSize, setBreakdownSize] = useState(0);
```

Comment:
@pieterlukasse : FYI: These variables were initially declared as undefined, then boolean conditionals (lines 132-140) set them to zero if false. I updated the state to initialize them at zero and removed the conditional.

Reply:
Per discussion this morning, the declarations are now initialized to null